brainstormforce/astra
# AST-1436 - Filter to disable all Meta Settings of Page/Post by default is not working in Gutenberg based metabox.

### Description
- Filter to disable all meta settings is not working in react based metabox
- https://wpastra.com/docs/disable-all-meta-settings-of-page-post-by-default/

### Types of changes
- Bug fix

### How has this been tested?
- Add above doc's filter in child theme
- Verify on post/page, the all default disable checkboxes should enable with filter

### Checklist:
